runtime-manager/modules/ROOT/pages/rtm-traffic-inspection.adoc

```diff
@@ -0,0 +1,111 @@
+= Traffic Inspection for Standalone Mules
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: agent, runtime manager, traffic inspection, standalone
+:page-deployment-options: hybrid
+
+The Traffic Inspection feature for Mules adds support in the agent for a forward proxy that is deployed in customers' premises and acts as a man in the middle between the Mule and the control plane by intercepting and inspecting all HTTPS traffic. 
```

Comment:
Global: Mules -> Mule instances or simply Mule.

Reply:
This sentence is quite complicated. Suggest breaking into two. Also, should "agent" be "Runtime Manager agent", since this is the first mention?

Reply:
```suggestion
The Traffic Inspection feature for Mule instances adds support to Runtime Manager agent for a forward proxy that is deployed in your environment. This proxy acts as a man-in-the-middle between Mule and the control plane, intercepting and inspecting all HTTPS traffic. 
```